pkg/synopsysctl/cmd_update.go

```diff
@@ -210,38 +205,35 @@ func updateAlertHelmBased(cmd *cobra.Command, alertName string, customerReleaseN
 			log.Fatalf("failed to read certificate file: %+v", err)
 		}
 		customCertificateSecretName := "alert-custom-certificate"
-		customCertificateSecret, err = alertctl.GetAlertCustomCertificateSecret(namespace, customCertificateSecretName, certificateData, certificateKeyData)
+		customCertificateSecret := alertctl.GetAlertCustomCertificateSecret(namespace, customCertificateSecretName, certificateData, certificateKeyData)
 		util.SetHelmValueInMap(helmValuesMap, []string{"webserverCustomCertificatesSecretName"}, customCertificateSecretName)
+		if _, err := kubeClient.CoreV1().Secrets(namespace).Create(&customCertificateSecret); err != nil {
+			if k8serrors.IsAlreadyExists(err) {
+				if _, err := kubeClient.CoreV1().Secrets(namespace).Update(&customCertificateSecret); err != nil {
+					return fmt.Errorf("failed to update certificate secret: %+v", err)
```

Comment:
I'm not sure whether an above code will work even in Black Duck codes because during update, we need to get the latest information from the cluster and update the values.. Can you please test this scenario by passing different certificates during update to make sure that it works without any error?

Reply:
in case, you find any issue, create a separate PR Michael